### ISSUE TITLE
Insufficient CUDA driver version error on CPU-only system 

### ISSUE THREAD

**Issue by ioangatop** [triton-inference-server/dali_backend#63]
Hi! I am using dali_backend as a image pre-processing model in my pipeline. While when deploying on a system with GPUs it works as expected, when I launch the server in a CPU-only system I have the following error; `dali_backend.cc:438 | CUDA runtime API error cudaErrorInsufficientDriver (35)`.

Here is my pipeline:
```
def export_pipeline(
    input_name='DALI_INPUT_0',
    image_dim=[800, 800],
    batch_size=1,
    num_threads=1,
    device='cpu',
    padding=True,
    device_id=None,
    output_dir='./out/',
):

    pipeline = dali.pipeline.Pipeline(
        batch_size=batch_size,
        num_threads=num_threads,
        device_id=device_id
    )

    with pipeline:
        data = fn.external_source(name=input_name, device=device)

        images = fn.image_decoder(data, device=device)
        images = fn.resize(images, size=image_dim, mode="not_larger", max_size=image_dim)
        images = fn.pad(images, fill_value=0, shape=[image_dim[0], image_dim[1], 1])

        input_shape = fn.shapes(images)
        input_shape = fn.cast(input_shape, dtype=dali.types.FLOAT)

        images = fn.transpose(images, perm=[2, 0, 1])
        images = fn.cast(images, dtype=dali.types.FLOAT)

        shapes = fn.peek_image_shape(data)
        shapes = fn.cast(shapes, dtype=dali.types.FLOAT)

        out = [
            images,
            input_shape,
            shapes
        ]
        pipeline.set_outputs(*out)
```

Thank you!

**Comment by szalpal** [comment]
Hello @ioangatop !

Thank you for your report. I confirm I was able to reproduce your problem. We'll take care of it asap.

**Comment by szalpal** [comment]
@ioangatop ,

I've merged the bugfix. It's going to be included in `tritonserver:21.06-py3`. Should you like to try it out beforehand, please build dali_backend from upstream, following [docker build instructions](https://github.com/triton-inference-server/dali_backend#docker-build-)

**Comment by ioangatop** [comment]
Thanks @szalpal!
